README.md

```diff
@@ -41,15 +60,27 @@ go_test(
 )
 ```
 
-You can define third party code using `go_get`:
+Compile binaries with `go_binary`:
+
+```python
+subinclude("///go//build_defs:go")
+
+go_binary(
+    name = "bin",
+    srcs = ["main.go"],
+)
+```
+
+Introduce dependencies on third-party Go modules with `go_repo`:
```

Comment:
Worth a reference to puku here?

Reply:
Worth doing, but `puku` is not quite my friend yet: I still routinely fall back to `go_module` and find myself fighting `puku sync` with some degree of regularity.  The idealized end-to-end workflow for "day in the life of an engineer using a repo with `plz`" isn't well defined and requires a fair amount of digging.